15_perms_lab/permslab.txt

```diff
@@ -2,14 +2,33 @@
 
   - change permission of file src/main.c to r--r--r-- using chmod letter notation
 
+chmod u=r,g=r,a=r src/main.c
+
   - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
 
+mews2659 ~/wildcards >> chmod 474 include/player.h 
+
   - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
 
+mews2659 ~/wildcards >> chmod 600 Doc/*.*
```

Comment:
Prefer `*` over `*.*` as not all files have a period in them. Better to use `chmod 600 Doc/*`